src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -1,10 +1,24 @@
 {% load static i18n %}
 
 <p>
-    {% blocktranslate %}Here is a summary of the emails that failed to send yesterday:{% endblocktranslate %}
+    {% blocktranslate %}Here is a summary of the failed procedures in the past 24 hours:{% endblocktranslate %}
 </p>
-<ul>
-    {% for log in logs %}<li>- Email for the event "{{ log.event }}" for submission {{ log.submission_uuid }}.</li>
-    {% endfor %}
-</ul>
 
+{% if failed_emails %}
+    <h5>{% trans "Emails" %}</h5>
+    <ul>
+        {% for email in failed_emails %}
+            <li>- Email for the event "{{ email.event }}" for submission {{ email.submission_uuid }}.</li>
+        {% endfor %}
+    </ul>
+{% endif %}
```

Comment:
Maybe using [`for ... empty`](https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#for-empty), we could show a default message?

Reply:
if there's nothing to report, don't include necessary bloat in the email. If you want people to read the emails, keep them as short as possible.